.github/ISSUE_TEMPLATE/chore.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Chore
+about: A template to capture non feature / enhancement work we need to do like work
+  to support other functions or housekeeping etc.
+title: ''
+labels: ux & visual design
```

Comment:
```suggestion
labels: ux-and-visual-design
```

I have updated the label to remove spaces to avoid issues with the add issue to project GH Action. This suggestion aligns to that.